samples/maps/demo/geoheatmap-equalearth/demo.js

```diff
@@ -0,0 +1,77 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@af1d7a1727/samples/data/geoheatmap-aerosol-dataset.json', function (data) {
+        Highcharts.mapChart('container', {
+
+            chart: {
+                map: topology,
+                backgroundColor: '#000',
+                animation: false
```

Comment:
Should we disable `animation`? It has an impact, when clicking plus/minus map navigation buttons and there is no animation during this zooming.

Reply:
We disabled the animation due to map zooming animation bug which cause the "moving" effect when zooming. But I tested it again and doesn't look so bad. Since we are going to fix this bug, I think we can enable this option.